package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ilp-protocol-stream",
-  "version": "2.4.0",
+  "version": "2.5.0",
```

Comment:
Why a minor version bump instead of a patch?

Also, usually I bump the version after the PR is merged (with `npm version patch|minor|major`) followed by `git push origin master --follow-tags`, to ensure that the release is tagged.

Reply:
:+1: I can move the version change from the pr.
I was thinking minor bump primarily due to receipts changes.